libs/facilities/frontend/view/src/lib/components/detail/detail.page.ts

```diff
@@ -171,4 +172,8 @@ export class XdDetailPage implements OnInit {
 			this.locked$.next(!this.locked$.getValue());
 		});
 	}
+
+	goBack() {
```

Comment:
Pls do it as a directive for the button as described here in the one comment with about 100 likes 

https://stackoverflow.com/questions/35446955/how-to-go-back-last-page

Reply:
I did it with a directive now. I was not sure where to put it. I put it in the facilities lib in a new folder "directives"